base-notebook/binder/environment.yml

```diff
@@ -10,7 +10,9 @@ dependencies:
   - bokeh
   - dask>=2.0.0
   - dask-kubernetes>=0.9.0
-  - nbserverproxy
+  - jupyter-server-proxy
   - ipywidgets
   - graphviz
-  - dask-labextension==0.3.3
+  - pip
+  - pip:
+    - git+https://github.com/ian-r-rose/dask-labextension.git@de-asyncify#egg=dask_labextension
```

Comment:
since we inherit from the repo2docker [base environment](https://github.com/jupyter/repo2docker/blob/master/repo2docker/buildpacks/conda/environment.yml) we can simplify this to:

```
# packages in addition to repo2docker/buildpacks/conda/environment.yml
name: pangeo
channels:
  - conda-forge
dependencies:
  - jupyterlab=1
  - dask=2
  - dask-kubernetes=0.9
  - tornado=6
  - jupyter-server-proxy
  - bokeh
  - graphviz
  - pip
  - pip:
    - git+https://github.com/ian-r-rose/dask-labextension.git@de-asyncify#egg=dask_labextension
```